graphs/possible_bipartition.py

```diff
@@ -1,12 +1,34 @@
 # Can be used for BFS
 from collections import deque 
 
+
 def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(N+E)
+        Space Complexity: O(N+E)
```

Comment:
🪐 Space complexity would actually be O(N) here since the two data structures you are creating are a graph that will hold an integer value for each of N nodes in `dislikes` and the queue will similarly hold at most `N` nodes.